stories/Collection.stories.js

```diff
@@ -65,6 +66,30 @@ export const flatStructure = () => (
   </DataProvider>
 );
 
+export const Ruler = () => (
```

Comment:
nit: put this below the `flatStructure` thing below.

Reply:
Oh oops.. fixed